runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -479,9 +479,26 @@ TR_J9ServerVM::classHasBeenReplaced(TR_OpaqueClassBlock *clazz)
 bool
 TR_J9ServerVM::classHasBeenExtended(TR_OpaqueClassBlock *clazz)
    {
+   uintptrj_t classDepthAndFlags = 0;
    JITaaS::J9ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
-   stream->write(JITaaS::J9ServerMessageType::VM_classHasBeenExtended, clazz);
-   return std::get<0>(stream->read<bool>());
+   bool dataFromCache = JITaaSHelpers::getAndCacheRAMClassInfo((J9Class *)clazz, _compInfoPT->getClientData(), stream, JITaaSHelpers::CLASSINFO_CLASS_DEPTH_AND_FLAGS, (void *)&classDepthAndFlags);
+   if(!clazz)
+      return false;
+   
+   if(dataFromCache)
+      {
+      // Check if flag is set, since once set it can not be removed
+      if((classDepthAndFlags & J9AccClassHasBeenOverridden) != 0)
```

Comment:
Our code convention is that if either the`if` of the `else` part have brackets, the other one must have too.

Reply:
Sounds good, I will change that.